spec/util/e2e-test.ts

```diff
@@ -7,7 +7,10 @@ import { IrcServer } from "../../src/irc/IrcServer";
 import { MatrixClient } from "matrix-bot-sdk";
 import { TestIrcServer } from "matrix-org-irc";
 import { IrcConnectionPool } from "../../src/pool-service/IrcConnectionPool";
+import { expect } from "@jest/globals";
 import dns from 'node:dns';
```

Comment:
Though this isn't a new line, my IDE is giving a warning for it:
> Module '"node:dns"' has no default export.

Using a namespace import should be a viable alternative:
```suggestion
import * as dns from 'node:dns';
```